cypress/support/commands/parent-commands.ts

```diff
@@ -108,6 +108,10 @@ Cypress.Commands.add('visualRegressionTestUrl', ({url, testName}) => {
       ...DEFAULT_VRT_OPTIONS,
     };
 
+    // Freeze the date for any date time diffs
+    const now = new Date(2018, 1, 1);
```

Comment:
question: what does this fix? Mind adding that reference inline in a comment for future maintainers please?

Reply:
Comment updated